### PR TITLE
refactor: Use constant instead of enum to reduce package size

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [X] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Use constant instead of enumeration to reduce package size |
| 🇨🇳 Chinese |  使用常量替代枚举减小包体积  |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/Base/Ellipsis.tsx [hunk 1]
```diff
@@ -118,10 +114,12 @@ export interface EllipsisProps {
...
-const STATUS_MEASURE_NONE = 0;
-const STATUS_MEASURE_START = 1;
-const STATUS_MEASURE_NEED_ELLIPSIS = 2;
-const STATUS_MEASURE_NO_NEED_ELLIPSIS = 3;
+enum STATUS_MEASURE {
+  NONE = 0,
+  START = 1,
+  NEED_ELLIPSIS = 2,
+  NO_NEED_ELLIPSIS = 3,
+}
```
Comment (afc163) [inline components/typography/Base/Ellipsis.tsx:122]: 这样改体积会变大，不划算。

Comment (kiner-tang) [reply components/typography/Base/Ellipsis.tsx:122]: 当时就是因为用枚举类型会导致体积变大才不用枚举的吗？

Comment (li-jia-nan) [reply components/typography/Base/Ellipsis.tsx:122]: 枚举会编译成 IIFE，返回一个对象，还不如用常量

Comment (kiner-tang) [reply components/typography/Base/Ellipsis.tsx:122]: 确实如果导致体积增大就不划算了

Comment (li-jia-nan) [reply components/typography/Base/Ellipsis.tsx:122]: 我记得 Affix 里面有枚举，可以顺手改成常量

Comment (kiner-tang) [reply components/typography/Base/Ellipsis.tsx]: 我全局搜一下看看

**Review by li-jia-nan — APPROVED** [review]

**Comment by li-jia-nan** [comment]
不知道有没有 break change，不太敢合，我先点个 ap，等大佬们合吧

**Review by zombieJ — APPROVED** [review]

**Comment by fpsqdb** [comment]
如果不需要遍历枚举的话，用`const enum`会不会更好

**Pull request merged**